frontend/src/components/dashboard/utils/charts.jsx

```diff
@@ -20,19 +20,38 @@ import {
   JOB_AGG_FILE_MIMETYPE_URI,
   JOB_AGG_OBS_NAME_URI,
   JOB_AGG_FILE_MD5_URI,
+  JOB_AGG_STATUS_URI_ORG,
+  JOB_AGG_TYPE_URI_ORG,
+  JOB_AGG_OBS_CLASSIFICATION_URI_ORG,
+  JOB_AGG_FILE_MIMETYPE_URI_ORG,
+  JOB_AGG_OBS_NAME_URI_ORG,
+  JOB_AGG_FILE_MD5_URI_ORG,
 } from "../../../constants/apiURLs";
 
 // constants
 const colors = getRandomColorsArray(10, true);
 
 // bar charts
 
-export const JobStatusBarChart = React.memo(() => {
+export const JobStatusBarChart = React.memo((props) => {
   console.debug("JobStatusBarChart rendered!");
+  /* eslint-disable */
+  var sendURL = JOB_AGG_STATUS_URI;
+  const porp = props.myprop;
+  const getValue = porp.key;
+  console.debug("props is ", getValue);
+  /* eslint-enable */
+
+  if (getValue) {
+    /* eslint-disable */
+    sendURL = JOB_AGG_STATUS_URI_ORG;
+    console.debug("this is sending to my urls", sendURL);
+    /* eslint-enable */
+  }
```

Comment:
more generally, I don't think it is necessary to create new API endpoints for this case. We should use the URL parameters to customize the results. This would avoid a lot of additional code that would be really difficult to maintain.

Example:
aggregate/status -> like it is right now
aggregate/status?org=True -> with the toggle enabled

Reply:
Yeah I was thinking the same. I just tried some sub-optimised code to check the core functionality. I will improve the code quality.